docs/getting-started/cli-getting-started.md

```diff
@@ -119,12 +134,14 @@ For more information, see [Writing scripts](../user-guide/cli-using-writing-scri
 
 ## Next steps
 
-You successfully installed Zowe CLI, issued your first commands, and wrote a simple script! Next, you might want to perform the following tasks:
+You successfully installed Zowe CLI, issued your first commands, and wrote a simple script! 
+
+Next, you might want to perform the following tasks:
 
 - Issue the `zowe --help` command to explore the product functionality, or review the online [web help](../user-guide/cli-using-displaying-help.md).
```

Comment:
Should we also mention `zowe --help-web`?
Or at least point them to the online web help?
- https://docs.zowe.org/stable/web_help/index.html

Reply:
This is a quick start guide, this doesn't have to be comprehensive. I think it's ok to omit both these things.